src/order-requests/order-requests.controller.ts

```
@@ -1,14 +1,158 @@
+    }
+
+    return this.orderRequestsService.rejectOrderRequest(orderRequestId, {
+      ...dto,
+      resolverId: user.id,
+    });
+  }
+  
   //TODO: /order-requests/{orderRequestId} (DELETE) 주문 요청 취소
+  @Delete(':orderRequestId')
+  async deleteOrderRequest(@Req() req, @Param('orderRequestId') orderRequestId: string) {
+  const user = req.user; // 요청한 사용자 정보 가져오기
```

ESLint

[error] Insert `··`

(prettier/prettier)